src/main/java/mate/academy/hibernate/relations/Main.java

```diff
@@ -16,17 +22,17 @@
         SessionFactory sessionFactory = HibernateUtil.getSessionFactory();
 
         Country usa = new Country("USA");
-        CountryService countryService = null; // TODO: initialize this instance
+        CountryService countryService = new CountryServiceImpl(new CountryDaoImpl(sessionFactory));
         countryService.add(usa);
 
         Actor vinDiesel = new Actor("Vin Diesel");
         vinDiesel.setCountry(usa);
-        ActorService actorService = null; // TODO: initialize this instance
+        ActorService actorService = new ActorServiceImpl(new ActorDaoImpl(sessionFactory));
```

Comment:
Ensure that the `ActorDaoImpl` class properly handles exceptions, as mentioned in the previous review.